idea/src/META-INF/plugin.xml

```diff
@@ -1520,6 +1520,15 @@
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.UnnecessaryJavaUsageInspection"
+                     displayName="Unnecessary java usage"
+                     groupName="Kotlin"
+                     enabledByDefault="true"
+                     cleanupTool="true"
+                     level="WARNING"
```

Comment:
This should be a WEAK_WARNING at most. There's nothing wrong with the original code - it's simply not as concise as it could be.